src/DiagPDF.cpp

```diff
@@ -0,0 +1,337 @@
+#include <ios>
+
+#include "AMReX_BLassert.H"
+#include "AMReX_Extension.H"
+#include "AMReX_GpuContainers.H"
+#include "AMReX_MultiFabUtil.H"
+#include "AMReX_ParmParse.H"
+#include "AMReX_SPACE.H"
+
+#include "DiagPDF.H"
+
+void DiagPDF::init(const std::string &a_prefix, std::string_view a_diagName)
+{
+	DiagBase::init(a_prefix, a_diagName);
+
+	amrex::ParmParse const hist_pp(a_prefix);
+	hist_pp.query("normalized", m_normalized);
```

Comment:
What does normalized do? I can't see where in the value of m_normalized is ever referenced again, and from what I can follow you are normalizing regardless of the value of this variable.

Reply:
Oops, that was a mistake. I will fix it.